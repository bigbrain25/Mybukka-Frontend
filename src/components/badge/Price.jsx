import React from 'react';
import PropTypes from 'prop-types';

import './price.scss';

const Price = ({ price }) => <h4 className="price-badge">â‚¦{price}.00</h4>;

export default Price;

Price.propTypes = {
  price: PropTypes.number.isRequired
};
