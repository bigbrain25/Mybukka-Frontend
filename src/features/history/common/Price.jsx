import React from 'react';
import PropTypes from 'prop-types';

const Price = ({ price }) => (
  <div className="history-price">
    <h5 className="text-muted">â‚¦{price}</h5>
  </div>
);

export default Price;

Price.propTypes = {
  price: PropTypes.string.isRequired
};
