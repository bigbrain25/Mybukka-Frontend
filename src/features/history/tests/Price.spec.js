import React from 'react';
import Price from '../common/Price';

describe('Price component', () => {
  const props = {
    price: '20000'
  };

  const wrapper = mount(<Price {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
