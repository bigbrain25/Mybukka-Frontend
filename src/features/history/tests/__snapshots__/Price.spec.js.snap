// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price component renders properly 1`] = `ReactWrapper {}`;
